src/main/java/org/zalando/nakadi/repository/db/EventTypeCache.java

```diff
@@ -37,6 +40,21 @@ public EventTypeCache(final EventTypeRepository eventTypeRepository, final Curat
         this.eventTypeCache = setupInMemoryEventTypeCache(eventTypeRepository);
         this.validatorCache = setupInMemoryValidatorCache(eventTypeCache);
         this.cacheSync = setupCacheSync(zkClient);
+        preloadEventTypes(eventTypeRepository);
+    }
+
+    private void preloadEventTypes(final EventTypeRepository eventTypeRepository) {
+        final long start = System.currentTimeMillis();
+        eventTypeRepository.list().stream().map(EventType::getName).forEach((name) -> {
+            try {
+                this.getEventType(name);
```

Comment:
Is it possible to provide a solution that will not get event-types from DB one-by-one?